src/test/java/com/example/universe/simulator/entityservice/integration/GalaxyIntegrationTest.java

```diff
@@ -76,6 +77,8 @@ void testGet() throws Exception {
         // then
         GalaxyDto result = readResponse(response, GalaxyDto.class);
         assertThat(result).isEqualTo(galaxy1);
+        GalaxyDto fromCache = modelMapper.map(cacheManager.getCache(GALAXY_CACHE_NAME).get(result.getId()).get(), GalaxyDto.class);
```

Comment:
Intellij is complaining about possible null pointers: `Method invocation 'get' may produce 'NullPointerException'`. Please add any necessary checks. You can use Optional in such cases.

Example:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/16168235/176372272-c5e51e47-ed65-4919-a895-b57fc0d21d1f.png">

Reply:
Surrounded such cases with Optionals.